Describe the Form - Customize Items demo (#659)
DevExpress/devextreme-demos: JSDemos/Demos/Form/CustomizeItem/description.md

> You can use the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array to configure all form items. This array can contain strings ([formData](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#formData) field names) and objects (item configurations). Use a string to create a simple item with default configuration as shown for `Email` item. 

Reviewer: I would move the last sentence to a separate paragraph. That way you'll have three paragraphs / "pieces of information":

- Paragraph 1: what's the entry point for configuration, what are the two types. 
- Paragraph 2: first type (simple string)
- Paragraph 3: second type (objects) 

Right now you describe two types, then list the first, and then the thought breaks off.

> You can use the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array to configure all form items. This array can contain strings ([formData](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#formData) field names) and objects (item configurations). Use a string to create a simple item with default configuration as shown for `Email` item. 

Reviewer 2: ```suggestion
You can use the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array to configure all form items. This array can contain strings ([formData](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#formData) field names) and objects (item configurations). Use a string to create a simple item with default configuration as shown for the `Email` item. 
```

To change the default settings, declare an item configuration object. Use the [dataField](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#dataField) property to bind an item to a field in the **formData** object. Use the [editorType](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#editorType) property to specify an item's data editor or configure the editor in the [editorOptions](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#editorOptions) object. You can also specify any other properties described in the [SimpleItem](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/) section.

> This demo shows how to specify **editorOptions**, **editorType**, [validationRules](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#validationRules), and [colSpan](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#colSpan) properties for the Form simple items.

Reviewer: The article with complicated noun phrase in plural doesn't read well to me. Both possessives and articles with plural are tricky and sometimes it's better to break a noun phrase apart and introduce a preposition (even "of"), just to be clear. 

```suggestion
This demo shows how to specify **editorOptions**, **editorType**, [validationRules](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#validationRules), and [colSpan](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#colSpan) properties for simple items in a Form widget.
```

> This demo shows how to specify **editorOptions**, **editorType**, [validationRules](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#validationRules), and [colSpan](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#colSpan) properties for simple items in a Form widget.

Reviewer 3: ```suggestion
This demo shows how to specify **editorOptions**, **editorType**, [validationRules](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#validationRules), and [colSpan](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#colSpan) properties for simple items in a Form component.
```
